libraries/shared/factories/storage/keys_loader.go

```diff
@@ -18,11 +18,11 @@ package storage
 
 import (
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/makerdao/vulcanizedb/libraries/shared/storage/utils"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage"
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
 type KeysLoader interface {
-	LoadMappings() (map[common.Hash]utils.StorageValueMetadata, error)
+	LoadMappings() (map[common.Hash]storage.StorageValueMetadata, error)
```

Comment:
Can probably rename this to just `ValueMetadata` if it lives in the `storage` pkg